examples/capture/main.rs

```diff
@@ -27,20 +29,37 @@ async fn run() {
         .await
         .unwrap();
 
-    // Rendered image is 256×256 with 32-bit RGBA color
-    let size = 256u32;
+    let args: Vec<_> = env::args().collect();
+    let (width, height) = match args.len() {
+        // 0 on wasm, 1 on desktop
+        0 | 1 => (100usize, 200usize),
+        3 => (args[1].parse().unwrap(), args[2].parse().unwrap()),
+        _ => {
+            println!("Incorrect number of arguments, possible usages:");
+            println!("*   0 arguments - uses default width and height of (100, 200)");
+            println!("*   2 arguments - uses specified width and height values");
+            return;
+        }
+    };
+
+    // It is a webgpu requirement that BufferCopyView.layout.bytes_per_row % 256 == 0
+    // So we calculate an appropriate padded_bytes_per_row that fits that requirement
+    let bytes_per_pixel = size_of::<u32>();
+    let real_bytes_per_row = width * bytes_per_pixel;
+    let padded_bytes_per_row_padding = (256 - (real_bytes_per_row) % 256) % 256;
```

Comment:
The set of parentheses on `real_bytes_per_row` don't seem to be affecting order of operations in any way. Additionally is there any way to simplify/explain this expression as it's very unclear what is going on at first glance, largely in part to the double mod. I thought of `((real_bytes_per_row + 255) / 256) * 256` but that's not terribly clearer either.

Reply:
yeah ... I agree it is weird looking. I had to lookup how to do it efficiently after kvark told me `align - real_bytes_per_row % align` was inefficient when `real_bytes_per_row == 0`.

Would including a link to https://en.wikipedia.org/wiki/Data_structure_alignment#Computing_padding be sufficient?

Reply:
I would just add something like

```rust
// Round size up to the next multiple of COPY_BYTES_PER_ROW_ALIGNMENT
```

to make things clear even if they can't understand the math.

Reply:
Thoughts?